projects/step-core/src/lib/pipes/plan-name.pipe.ts

```diff
@@ -0,0 +1,11 @@
+import { Pipe, PipeTransform } from '@angular/core';
+import { Plan } from '../step-core.module';
+
+@Pipe({
+  name: 'planName',
+})
+export class PlanNamePipe implements PipeTransform {
+  transform(plan: Plan): string | undefined {
+    return plan.attributes?.['name'];
+  }
+}
```

Comment:
Is this pipe used somewhere?

Reply:
yes, I am using it in the EE version (there is a corresponding branch over there too)